lib/r2d2/version.rb

```diff
@@ -1,3 +1,3 @@
 module R2D2
-  VERSION = "1.0.0" unless defined? R2D2::VERSION
+  VERSION = "1.0.1" unless defined? R2D2::VERSION
```

Comment:
This should probably be a minor version bump (to 1.1.0) or major (2.0.0) since it's adding support for ECv2.